pkg/controller/topo/entity/reconciler.go

```diff
@@ -29,7 +29,6 @@ import (
 
 var log = logging.GetLogger("controller", "topo", "entity")
 
-const topoService = "onos-topo"
 const topoFinalizer = "topo"
```

Comment:
We should namespace this finalizer, something like `topo.onosproject.org/entity` so it can't conflict with other systems

Reply:
@kuujo Does it mean to add the entity's namespace to finalizer?